### ISSUE TITLE
Feature Request: Launch Images in New Tab (or Window)

### ISSUE THREAD

**Issue by AA7US** [swharden/QRSSplus#7]
When clicking on images of a grabber's history, would it be possible to have them launch in a new browser tab or window instead of the current tab?

Thank you and 73,

John
AA7US

**Comment by swharden** [comment]
> When clicking on images of a grabber's history, would it be possible to have them launch in a new browser tab or window instead of the current tab?

Sounds good - I'll make that change later today and close this issue when it's done. In the mean time, middle-clicking the images will probably open them in a new window.

note to self: this is the line to modify to change this behavior
https://github.com/swharden/QRSSplus/blob/a30955c221c02fcd5bac47c3d490f2d3bbb51748/qrssPlus.js#L143

**Comment by AA7US** [comment]
Thank you Scott! 👍🏼

**Comment by swharden** [comment]
I made the change! Historical grabs now open in new windows when clicked. 111941bd519e710b410241589f751dd646e113af It's live on the website: http://swharden.com/qrss/plus/

I decided to leave all the other links on the page as they are. Users can middle-click any link to open it in a new window, or left-click a link to open them in the same window. I want the user to be able to decide how they want to navigate: if we use code to _force_ links to open in new windows, we take this decision away from the user.
